src/index.html

```diff
@@ -1,14 +1,30 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <h1>Stop watch</h1>
+    <div class="watch">
+      <div class="hour"></div>
+
+      <div class="minute">
+        <div class="min"></div>
+        <div class="min min--invisible"></div>
+      </div>
+
+      <div class="second">
+        <div class="sec"></div>
+        <div class="sec sec--invisible"></div>
+      </div>
```

Comment:
it's a not very good decision. Can you make it and style this structure?
and **use BEM**

```suggestion
      <div class="minute"></div>
      <div class="second"></div>
```

Reply:
мне так проще было центрировать это все.

Reply:
проще - не значит лучше, другому человеку будет сложно разобраться в таком коде

Reply:
там вращается две стрелки, одна из них невидимая, центр автоматически совмещается в центром других стрелок в watch - назависимо от толщины и длины. мне показалось это очевидным и простым решением.
https://i.imgur.com/lyFZckh.png